docs/whitepaper.tex

```diff
@@ -157,80 +157,83 @@ \section{Implementation details}
-}, we need a liquidity token to exist, with a minting policy linked to a
-specific pool.
-Secondly, as in the Cardano Application Template, we want each pool be
-identified by an NFT minted through a specific minting policy. This allows
-anyone to verify that a UTXO at the pool script address is valid, as well as to
-trace each pool.
-Finally, our protocol contains a number of cyclic dependencies between the
-different scripts. In order to resolve this, we follow the Config UTXO pattern.
+}, we need a liquidity token linked to the respective pool to exist. 
+This implies that the liquidity token needs to be minted/ burnt using a minting policy linked to a specific pool.
+Secondly, as in the Cardano Application Template, we want the liquidity pools to be
+identifiable, traceable and verifiable (valid or not).
+We achieve this by minting an identifier NFT which requires its own minting policy.
+Lastly, our protocol consists of a number of scripts whose functionalities are dependent
+on each other, where some of those functional dependencies are cyclic. We model/ lock these
+functional dependencies using the \textit{Config UTxO} pattern.
 
 \subsection{Config UTXO}
 
-As mentioned above, our protocol will contain cyclic dependencies between the
-different scripts. Imagine we go about the situation naively, and we make each
-script dependent on the others. Now the hash of each script is dependent on the
-hash of the other scripts, which leads to a computationally very difficult
-problem to resolve. Therefore, we propose to use a different solution: The
+As mentioned above, our protocol consists of a number of scripts whose functionalities are dependent on each other,
+where some of those functional dependencies are cyclic.
+A naive solution to model/ lock these functional dependencies would be to 
+make the hash of one script a parameter of the other script and thus a dependency of the hash of the other scripts.
+This approach is difficult to implement, as the realation is cyclic.
+We propose to use a different solution to this problem: The
 Config UTXO. \\
 
-The idea is that a Config UTXO is built, identified by an NFT, which holds in
-its datum the hashes of all the scripts involved in the protocol. Each script
-can then refer to the hashes in the Config UTXO in order to verify the other
-scripts. In our case, the Config UTXO will only contain an NFT (to identify
-itself) and a datum with the liquidity token
-minting policy hash, and the pool validator script address. \\
+The Config UTXO is a dedicated UTxO which is identified by a NFT, its datum 
+keeps track of all the script hashes involved in the protocol. Each script
+is then able refer to the script hashes in the Config UTXO in order to verify the other
+scripts. In our case, the Config UTXO will contain the following data in the datum: 
+the liquidity token minting policy hash and the pool validator script address. \\
```

Comment:
Could we add some explanation here about why the `Pool ID Minting Policy Hash` is missing from this?

Reply:
Yes please someone explain, I just took what was there and tried to make more sense out of it

Reply:
The pool id token minting policy is the only validator reading the config utxo and it can get its own hash from the script context so it doesn't need to be stored in the config utxo.